src/app/portlets/dot-edit-page/layout/components/dot-edit-layout-grid/dot-edit-layout-grid.component.ts

```diff
@@ -35,8 +31,13 @@ export class DotEditLayoutGridComponent implements OnInit, ControlValueAccessor
     @ViewChild(NgGrid)
     ngGrid: NgGrid;
 
+    @ViewChild('classToAddInput')
+    classToAddInput: ElementRef;
```

Comment:
here I want to do in the html something like:

<input pInputText #classToAddInput type="text" [(value)]=''classToAdd"/>

and here something like:

classToAdd: string;

I don't know why but not work in that way,  but I have to try it again.